tests/TestCompose_dealii-of.Ubuntu1604.home/.env

```diff
@@ -1,4 +1,3 @@
 OPENFOAM_TAG=latest
 DEALII_TAG=latest
-SYSTEST_REMOTE=precice/
```

Comment:
Line here and below could be kept, no need to delete it. It does not influence the possibility of repo specification above.